healthScore/user_utils.py

```diff
@@ -90,3 +92,16 @@ def get_health_history_details(request):
 
         zipped_details = zip(detailed_history_list, each_details)
         return zipped_details
+
+
+def hash_password(password):
+    # Generate a salt
+    salt = bcrypt.gensalt()
```

Comment:
Can you check with Jasmine regarding this? I think Django itself hashes the passwords (I'm not sure if it's ONLY for Users or anything in general).

Reply:
Currently it is for the `Users` model only in our application. In order to automatically generate the hash, like in users model, we will have to inherit the `hospitals` model from **AbstractBaseUser** class and create ** CustomUserManager**. Then migrations need to be generated and updated for the same.

Since we only need to hash the password and nothing else, will it be okay if we use custom hashing for it?

Reply:
Ohh alright. Let's leave it like this in that case

Reply:
Thanks